operator/config.go

```diff
@@ -9,45 +9,73 @@ import (
 )
 
 type Config struct {
+	// The address for the operator
 	OperatorAddress string
 
-	// Avs Reader addresses
+	// These addresses are used to initialize the avs registry chain reader
 	OperatorStateRetrieverAddress string
 	ServiceManagerAddress         string
 	AVSRegistryCoordinatorAddress string
 
+	// The url exposed by the anvil node to call the contract methods
 	EthRpcUrl string
-	EthWsUrl  string
 
-	BlsPrivateKeyStorePath        string
+	// The url exposed by the anvil node to listen to the contract events
+	EthWsUrl string
+
+	// The path to the location of the bls private key on local storage
```

Comment:
Let's capitalize BLS
```suggestion
	// The path to the location of the BLS private key on local storage
```

Reply:
Done in [ca84d8b](https://github.com/Layr-Labs/eigensdk-go/pull/692/commits/ca84d8b767c1539ffec12178dc1070c286a362b5)